lib/time/doc.go

```diff
@@ -0,0 +1,56 @@
+/*Package time defines time primitives for starlark, based heavily on the time
+package from the go standard library.
+
+  outline: time
```

Comment:
What is this notation here?

Can we brush up this comment so that it conforms to godoc norms? Ideally with a fixed-width table summarizing the operations, and then a more expansive prose explanation below.